app/modules/booking/viewCurrentBookings/viewCurrentBookings.query.js

```diff
@@ -1,25 +1,39 @@
 const { user_tickets: UserTickets, users: Users, tickets: Tickets, Sequelize } = require('../../../../models/index');
 const Op = Sequelize.Op;
-
-const viewCurrentBookingsQuery = () => {
-    return UserTickets.findAll({
-        attributes: [
-            ['id', 'booking_id'], 'ticket_id', 'quantity'
-        ],
-        where: {
-            isDeleted: {
-                [Op.eq]: 'false'
-            }
-        },
-        include: [{
-                model: Users,
-                attributes: ['id', 'firstName', 'lastName']
+const viewCurrentBookingsQuery = (page, pagesize) => {
+    try {
+        let query = {
+            attributes: [
+                ['id', 'booking_id'], 'ticket_id', 'quantity'
+            ],
+            where: {
+                isDeleted: {
```

Comment:
table column name should be in a standard form is_deleted, mix and match of the column name identified.